.github/workflows/e2e-tests-ondemand.yaml

```
@@ -3,7 +3,7 @@ name: E2E tests ondemand
 on:
   schedule:
     # every Mon/Wed/Fri at 08:00 UTC
```

please update or remove comment.
```suggestion
    # At 03:00 UTC on Monday, Wednesday, and Friday.
```